css/.sass-lint.yml

```diff
@@ -0,0 +1,350 @@
+        - outline-style
+        - outline-color
+        - filter
+        - null
+        - color
+        - font
+        - font-family
+        - font-style
+        - font-weight
+        - font-variant
+        - font-size
+        - font-size-adjust
+        - line-height
+        - letter-spacing
+        - word-spacing
+        - direction
+        - word-wrap
+        - overflow-wrap
+        - word-break
+        - hyphens
+        - white-space
+        - text-align
+        - text-align-last
+        - text-indent
+        - text-overflow
+        - text-transform
+        - text-decoration
+        - text-shadow
+        - text-rendering
+        - text-size-adjust
+        - vertical-align
+        - tab-size
+        - list-style
+        - list-style-position
+        - list-style-type
+        - list-style-image
+        - null
+        - transform
+        - transform-function
+        - transform-origin
+        - transform-style
+        - perspective
+        - perspective-origin
+        - backface-visibility
+        - null
+        - transition
+        - transition-property
+        - transition-duration
+        - transition-timing-function
+        - transition-delay
+        - animation
+        - animation-name
+        - animation-duration
+        - animation-timing-function
+        - animation-delay
+        - animation-iteration-count
+        - animation-direction
+        - animation-fill-mode
+        - animation-play-state
+        - null
+        - cursor
+        - user-select
+        - resize
+        - pointer-events
+        - null
+        - '-webkit-appearance'
+        - '-moz-appearance'
+        - '-webkit-overflow-scrolling'
+        - '-webkit-tap-highlight-color'
+        - '-webkit-text-stroke'
+  property-units:
+    - 0
+    - global:
+        - ch
+        - em
+        - ex
+        - rem
+        - cm
+        - in
+        - mm
+        - pc
+        - pt
+        - px
+        - q
+        - vh
+        - vw
+        - vmin
+        - vmax
+        - deg
+        - grad
+        - rad
+        - turn
+        - ms
+        - s
+        - Hz
+        - kHz
+        - dpi
+        - dpcm
+        - dppx
+        - '%'
+      per-property: {}
+  quotes:
+    - 0
+    - style: double
+  shorthand-values:
+    - 2
+    - allowed-shorthands:
+        - 1
+        - 2
+        - 3
+  single-line-per-selector: 2
+  space-after-bang:
+    - 2
+    - include: false
+  space-after-colon:
+    - 2
+    - include: true
+  space-after-comma:
+    - 2
+    - include: true
+  space-before-bang:
+    - 2
+    - include: true
+  space-before-brace:
+    - 2
+    - include: true
+  space-before-colon: 2
+  space-between-parens:
+    - 2
+    - include: false
+  trailing-semicolon: 2
+  url-quotes: 2
+  variable-for-property:
+    - 2
+    - properties:
+        - font
+  variable-name-format:
+    - 2
+    - allow-leading-underscore: true
+      convention: '[a-z]|-{3}|_{3}|[^_]_[^_]'
```

Comment:
@jeffkamo this regex is taken from our SCSS-lint `NameFormat` [rule](https://github.com/mobify/mobify-code-style/blob/develop/css/.scss-lint.yml#L132) which is actually lifted from the [SCSS-lint source code](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/selector_format.rb#L58). It's the same regex that SCSS-lint uses for hyphenated BEM format.